### PR TITLE
Created submissions & bounty dashboard page

### PR DESCRIPTION
**Pull request by jessiwright**
Per request from Clare & Cortex team

### CODE REVIEW

File: docs/programs/dashboards.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ id: "programs/dashboards"
 The Program Dashboard provides information on various metrics for your program. This helps you track the progress of your program and it also gives you a detailed summary and understanding of your program status.
 
 HackerOne provides dashboards on these pages:
-* [Submissions](/programs/submissions-dashboard.html)
```
Comment (martijnrusschen) [inline docs/programs/dashboards.md:10]: Are we removing the old page as well?

Comment (jessiwright) [reply docs/programs/dashboards.md:10]: yes, they said this was replacing it

Comment (martijnrusschen) [reply docs/programs/dashboards.md:10]: Ok, let's make sure we delete the old page in this PR as well.

Comment (jessiwright) [reply docs/programs/dashboards.md:10]: Good call, deleted the page

**Review by martijnrusschen — APPROVED** [review]

**Pull request merged**